lib/install/validate/storage.go

```diff
@@ -39,29 +39,39 @@ func (v *Validator) storage(ctx context.Context, input *data.Data, conf *config.
 		v.NoteIssue(err)
 		return
 	}
+	if err != nil {
```

Comment:
yup - just pass it straight through to NoteIssue and that'll do the right thing.